### PR TITLE
Setup graphql

### PR DESCRIPTION
**Pull request by bimal125**
## Changes

* Setup graphql server
* Added Country profile query

Mention related users here if any.

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] n+1 queries
- [x] flake8 issues
- [x] `print`
- [x] typos
- [x] unwanted comments

## This PR contains valid:

- [ ] tests
- [ ] permission checks (tests here too)

### CODE REVIEW

File: apps/visualization/types.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    vaccine_supply: auto
+    readiness: auto
+    vulnerability: auto
+    risk: auto
+    response: auto
+
+
+@strawberry.django.type(CountryProfile)
+class CountryListType(CountryProfileType):
+    pass
```
Comment (k9845) [inline apps/visualization/types.py:68]: Why?

File: apps/visualization/types.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
+# types.py
+import strawberry
+from strawberry import auto
+from .models import CountryProfile
+from strawberry.types import Info
+from typing import List
```
Comment (k9845) [inline apps/visualization/types.py:6]: Maintain order of import

File: apps/visualization/schema.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    except CountryProfile.DoesNotExist:
+        return None
+
+
+@strawberry.type
+class Query:
+    country_profiles: List[CountryProfileType] = strawberry.django.field()
+
+    @strawberry.field
+    def country_profile(self, iso3: Optional[str] = None) -> CountryProfileType:
```
Comment (thenav56) [inline apps/visualization/schema.py:22]: Let's make iso3 required.

File: apps/visualization/types.py (hunk 1 above)
Comment (thenav56) [inline apps/visualization/types.py:68]: May cause cache issue on apollo client.

**Review by thenav56 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**